Tasks/PyPIPublisher/task.json

```diff
@@ -32,6 +32,14 @@
       "label": "Python package path",
       "required": true,
       "helpMarkDown": "Python package directory to be published, where setup.py is present."
+    },
+    {
+        "name": "wheel",
+        "type": "boolean",
+        "label": "Upload wheel",
+        "defaultValue": "false",
+        "required": false,
+        "helpMarkDown": "Along with source distribution file, this will build and publish universal wheel (platform independent) of given Python package on PyPI, which will be named as {distribution}-{version}-py2.py3-none-any.whl"
```

Comment:
If checked, then task will additionally build and publish universal wheel (platform independent) of this package. For more information regarding wheel see here